spec/controllers/lectures_controller_spec.rb

```diff
@@ -139,18 +142,19 @@
 
   describe "DELETE #destroy" do
     before(:each) do
-      login_lecturer
+      # login user
+      @lecturer = FactoryBot.create(:user, :lecturer)
+      @lecture = FactoryBot.create(:lecture, lecturer: @lecturer)
+      sign_in(@lecturer, scope: :user)
```

Comment:
3 different ways of login lecturer? Is this usefull? I don't see the need of doing this differently?!